src/gamefalloutttw.cpp

```diff
@@ -266,7 +270,7 @@ QString GameFalloutTTW::binaryName() const
 
 QString GameFalloutTTW::gameShortName() const
 {
-  return "TTW";
+    return "FalloutNV";
```

Comment:
I understand the issue with LOOT and the TTW name, we should just confirm that this does not impact stuff elsewhere... From a quick look at MO2 main code, this would prevent users from using custom Splash Screen for TTW against Fallout NV for instance.

Reply:
From some quick research I didn't see anything out of the blue, thankfully there isn't a splash screen created yet for FNV or TTW

Reply:
The `lootGameName` or `sortGameName` might be a good idea because that would avoid mixing names for different purposes.